crates/wasmtime/src/config.rs

```diff
@@ -3024,7 +3024,10 @@ impl PoolingAllocationConfig {
 pub(crate) fn probestack_supported(arch: Architecture) -> bool {
     matches!(
         arch,
-        Architecture::X86_64 | Architecture::Aarch64(_) | Architecture::Riscv64(_)
+        Architecture::X86_64
+            | Architecture::Aarch64(_)
+            | Architecture::Riscv64(_)
+            | Architecture::S390x
     )
 }
```

Comment:
Mind going ahead and removing this function altogether?

Reply:
Sure, will do.